src/app/auth/login/login.component.ts

```diff
@@ -31,13 +29,11 @@ export class LoginComponent implements OnInit {
     this.loginForm.reset();
     this.authService.signin(formValue).subscribe((goodCredentials) => {
       if (goodCredentials) {
-        this.showErrorMessage = false;
-        this.router.navigate(['chat', '1']);
+        this.router.navigate(['conversations']);
         return;
       } else {
         alert("Username or password incorrect")
```

Comment:
this can probably be removed and replaced with adding this error to show errors instead of it being outside the if else statement

Reply:
this is in reference to line 35...im still not used to how comments are formatted